Tests/TuistGeneratorTests/Generator/ProjectDescriptorGeneratorTests.swift

```diff
@@ -335,6 +335,29 @@ final class ProjectDescriptorGeneratorTests: TuistUnitTestCase {
         XCTAssertEqual(pbxProject.developmentRegion, "de")
     }
 
+    func test_generate_localSwiftPackageGroup() throws {
+        // Given
+        let project = Project.test(
+            packages: [
+                .local(path: try AbsolutePath(validating: "/Packages/LocalPackageA")),
+                .local(path: try AbsolutePath(validating: "/Packages/LocalPackageB")),
```

Comment:
I would maybe use a different folder name rather than `Packages` to make clear that's not what defines the group name

Reply:
Renamed it to `/LocalPackages/...`, hope that is distinct enough 😄 We can of course rename if not.